pkgs/development/python-modules/datefinder/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, python-dateutil
+, pytz
+, regex
+}:
+
+buildPythonPackage rec {
+  pname = "datefinder";
+  version = "0.7.3";
```

```suggestion
  version = "0.7.3";
  format = "setuptools";
```
Would you please add `format`?

---

Done!